frontend/src/components/elements/ListViewElement/ListButtonElements/BuildingListButtonComponent.tsx

```diff
@@ -52,7 +52,8 @@ const BuildingListButtonComponent = (props: ListItemProps) => {
                             horizontal: 'right',
                         }}
                     >
-                        <Avatar src='public/media/img.png' alt="building" className={styles.image}/>
+                        <Avatar src={props.data.image ? props.data.image : 'public/media/img.png'} alt="building"
```

Comment:
Could you change `alt="building"` to `alt={props.data.name}`? This way, the the avatars in the sidebar will have the first letter of the building's name (instead of the b we have right now) in case public/media.img.png fails to load